public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -688,7 +694,7 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
         confirmText="Deactivate"
         icon="exclamation-triangle"
         onConfirm={() => {
-          setValue('editorSettings.simplifiedNotificationEditor', true);
```

Comment:
this fixes the bug not setting the switch to not advanced when confirming `deactivate`

Reply:
I'm guessing the change from `simplifiedNotificationEditor` to `simplifiedQueryEditor` is what fixes the issue and is intentional, right?

Reply:
correct